src/App.tsx

```diff
@@ -1,10 +1,37 @@
 /* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useRef, useState } from 'react';
 import { UserWarning } from './UserWarning';
 import { USER_ID } from './api/todos';
+import { Todo } from './types/Todo';
+import { Header } from './components/Header/Header';
+import { TodoList } from './components/TodoList/TodoList';
+import { getFiltredTodos } from './utils/getFiltredTodos';
+import { FilterStatus } from './types/FilterStatus';
+import { getTodos } from './api/todos';
+import { Footer } from './components/Footer/Footer';
+import { Error } from './types/Error';
+import cn from 'classnames';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [error, setError] = useState('');
+  const [selectedFilter, setSelectedFilter] = useState(FilterStatus.All);
+
+  const filtredTodos = getFiltredTodos(todos, selectedFilter);
+  const title = useRef<HTMLInputElement>(null);
+
+  useEffect(() => {
+    getTodos()
+      .then(res => setTodos(res))
+      .catch(() => setError(Error.UnableLoad))
+      .finally(() => {
```

Comment:
[Not critical]
No need to clear error for both successful and failure case. In case of success error would be already empty.